### PR TITLE
Fixes #23418: Release notes for 8.0

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23418

### CODE REVIEW

File: release-data/changelogs/8.0/index.adoc [hunk 1]
```diff
@@ -1,6 +1,140 @@
...
+Rudder 8.0 is still in beta, and *must not* be used in production for now.
+
+====
+
+We’re thrilled to announce the availability of pre-releases of Rudder 8.0.
+
+The major Rudder 8 version will be mainly dedicated to improving compliance
+features. This will affect different parts of the application.
+
+nutriscore
```
Comment (VinceMacBuche) [inline release-data/changelogs/8.0/index.adoc:16]: I guess that should be removed ;)

**Comment by amousset** [comment]
PR updated with a new commit

File: release-data/changelogs/8.0/index.adoc (hunk 1 above)
Comment (amousset) [reply release-data/changelogs/8.0/index.adoc]: done (sadly)

**Comment by amousset** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

File: release-data/changelogs/8.0/index.adoc [hunk 2]
```diff
@@ -1,6 +1,179 @@
...
+Rudder 8.0 is still in beta, and *must not* be used in production for now.
+
+====
+
+We’re thrilled to announce the availability of pre-releases of Rudder 8.0.
+The 8.X versions will be mainly dedicated to developing the compliance
+axis. This will affect different parts of the application, including the server
+and the agent, stay tuned for upcoming features!
+
+The 8.0 version is the first of the 8.X branch. As such, it updates the list of supported OSes:
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:16]: "The list of supported OSes changes to:" ?

File: release-data/changelogs/8.0/index.adoc [hunk 3]
```diff
@@ -1,6 +1,179 @@
...
+
+* Supported as Rudder server and relay: Debian 11 & 12, Ubuntu LTS 22.04, RHEL/CentOS/AlmaLinux/Rocky/Oracle 8 & 9, SLES 15 SP4+, Amazon Linux 2023
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:18]: * For Rudder server and relay, the supported OSes are now: Debian 11 & 12, Ubuntu LTS 22.04, RHEL/CentOS/AlmaLinux/Rocky/Oracle 8 & 9, SLES 15 SP4+, Amazon Linux 2023

File: release-data/changelogs/8.0/index.adoc [hunk 4]
```diff
@@ -1,6 +1,179 @@
...
+ ** As a consequence, the server is not supported anymore on: Debian 10, Ubuntu 20.04, SLES 15 SP3* and below, Amazon Linux 2
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:19]: For Rudder server and relay, these operating systems are not supported anymore: Debian 10, Ubuntu 20.04, SLES 15 SP3* and below, Amazon Linux 2

File: release-data/changelogs/8.0/index.adoc [hunk 5]
```diff
@@ -1,6 +1,179 @@
...
+image::images/kb.png[]
+
+=== ✅ Pre/post upgrade hooks for campaigns on Linux
+
+Campaigns now include a mechanism to run actions before or after the upgrade.
+This covers a variety of needs:
+
+* Run "pre-flight" check: available disk space check, backup check, etc... It allows stopping the update process if the system is not ready.
+* Run pre-update actions: take a VM snapshot of the system, etc.
+* Run post-update actions: update a CMDB, notify the user, etc.
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:43]: i thought there were a pre-reboot also ?

File: release-data/changelogs/8.0/index.adoc [hunk 6]
```diff
@@ -1,6 +1,179 @@
...
+      enforce: "true"
+----
+
+These two representations are equivalent, and you will be able to switch between both of them to edit the same technique.
+
+image::images/yaml.png[]
+
+The YAML format also becomes the internal storage format for techniques (used in the configuration
+git and archives).
+As this format will be stable, it will allow easy exchange of techniques between Rudder servers
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:79]: * This format is stable, so techniques can be shared between Rudder servers having different versions.

File: release-data/changelogs/8.0/index.adoc [hunk 7]
```diff
@@ -1,6 +1,179 @@
...
+running different versions.
+
+=== ⚙️ The `rudderc` toolbox
+
+The new format comes with a matching CLI too called `rudderc` (think "Rudder compiler").
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:84]: the "too" seems superflous

File: release-data/changelogs/8.0/index.adoc [hunk 8]
```diff
@@ -1,6 +1,179 @@
...
+It is responsible for compiling the policies for their target platform (into a low-level format
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:85]: yaml files are policies or techniques? i'm confused by this naming here

File: release-data/changelogs/8.0/index.adoc [hunk 9]
```diff
@@ -1,6 +1,179 @@
...
+minimal setups without a syslog daemon.
+
+=== 📜 The accepted/refused node history is now stored in the database
+
+It was previously on the filesystem (in `/var/rudder/inventories/historical`).
+This will be less confusing for users and allow easier management.
+
+=== 🥸 The mustache templating will fail in case of undefined variables (on Linux)
+
+Like already done for Jinja2 templating, the mustache implementation will also fail
```
Comment (ncharles) [inline release-data/changelogs/8.0/index.adoc:168]: "To match the Jinja2 templating implementation" ?

File: release-data/changelogs/8.0/index.adoc (hunk 5 above)
Comment (amousset) [reply release-data/changelogs/8.0/index.adoc:43]: it was left fort later

**Pull request merged**